datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validate_snmp_profiles.py

```diff
@@ -4,7 +4,7 @@
 from . import validators
```

Comment:
note: Hudi test is failing, but it's unrelated to this PR. Should be fixed by: https://github.com/DataDog/integrations-core/pull/10754